Dialog/TilesetEditDialog.cpp

```diff
@@ -1112,13 +1158,13 @@ void TilesetEditDialog::on_pushButton_ImportPalette_clicked()
                 this,
                 tr("Open palette file"),
                 romFileDir,
-                tr("usenti pal file (*.pal);;YY-CHR pal file (*.pal)"),
+                tr("usenti pal file (*.pal);;YY-CHR pal file (*.pal);;Raw Binary palette (*.bin)"),
                 &selectedfilter
     );
     if(qFilePath.isEmpty()) return;
 
     // Check the file extension
-    if(!qFilePath.endsWith(".pal", Qt::CaseInsensitive))
+    if((!qFilePath.endsWith(".pal", Qt::CaseInsensitive)) && (!qFilePath.endsWith(".bin", Qt::CaseInsensitive)))
     {
         QMessageBox::critical(this, QString("Error"), QString("Wrong file extension!"));
```

Comment:
Let's print the expected file extensions (.bin, .pal)

Reply:
:heavy_check_mark: